boards/arm/thingy53_nrf5340/thingy53_nrf5340_cpuapp_ns.yaml

```diff
@@ -15,3 +15,4 @@ supported:
   - usb_cdc
   - usb_device
   - netif:openthread
+  - gpio
```

Comment:
Perhaps it would be good to place it before `i2c` to keep consistency with `thingy53_nrf5340_cpuapp_ns.yaml` (the best would be to have all these entries in the alphabetical order but it's not a subject for this PR).

Reply:
Done